examples/multi-gpu-movielens/01-03b-MultiGPU-Download-Convert-ETL-with-NVTabular-Training-with-TensorFlow.ipynb

```diff
@@ -666,6 +668,16 @@
      "text": [
       "Writing ./hvd_wrapper.sh\n"
      ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "distributed.client - ERROR - Failed to reconnect to scheduler after 10.00 seconds, closing client\n",
```

Comment:
Error showing in render?

Reply:
Yeah, not sure where this one is coming from exactly. I think it's related to Dask Distributed and the cluster shutdown process, but it doesn't seem to reflect an issue that prevents the notebook from executing.